### PR TITLE
doc/cephfs/nfs: add ganesha configuration hierarchy

### PR DESCRIPTION
**Pull request by varshar16**
<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/cephfs/fs-nfs-exports.rst [hunk 1]
```diff
@@ -16,6 +16,48 @@ Requirements
...
+                        +----+-----+    +----+-----+    +-----+----+
+                             ^               ^                ^
+                             |               |                |
+                             +--------------------------------+
+                                        %url |
+                                             |
+                                    +--------+--------+
+                                    |                 |  rados://$pool/$namespace/conf-nfs.$svc
+                                    |  conf+nfs.$svc  |  (common config)
+                                    |                 |
```
Comment (batrick) [inline doc/cephfs/fs-nfs-exports.rst:43]: Where is the user specified config?

Comment (varshar16) [reply doc/cephfs/fs-nfs-exports.rst:48]: I have added it.

**Comment by sebastian-philipp** [comment]
jenkins render docs

**Review by sebastian-philipp — APPROVED** [review]

**Comment by sebastian-philipp** [comment]
https://ceph--41442.org.readthedocs.build/en/41442/cephfs/fs-nfs-exports/#ganesha-configuration-hierarchy

**Comment by varshar16** [comment]
ping @batrick

**Review by batrick — APPROVED** [review]

**Pull request merged**